src/style.css

```diff
@@ -1,3 +1,58 @@
 body {
+  --blue-color: #00acdc;
   margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 0 50px;
+}
+
+.header__logo {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  line-height: 60px;
```

Comment:
it's not the best solution to hardcode line-height, add vertical paddings to the links instead